src/bfx_tools/bedtools.ml

```diff
@@ -43,7 +43,7 @@ let bamtofastq
       (["-fq"; r1], r1, None)
   in
   let bedtools = Machine.get_tool run_with Machine.Tool.Default.bedtools in
-  let src_bam = input_bam#product#path in
+  let src_bam = sorted_bam#product#path in
```

Comment:
Can you also remove the `sorted_bam` value also?
(github does not let me comment on the non-diff line above)